packages/sdk-demo/src/hooks/useTheme.tsx

```diff
@@ -0,0 +1,36 @@
+import { useLocalStorage } from 'react-use';
+
+import { ThemeConfig } from '@/types';
+import {
+  materialDark as themeMaterialDark,
+  materialLight as themeMaterialLight,
+  moniteDark as themeMoniteDark,
+  moniteLight as themeMoniteLight,
+} from '@team-monite/sdk-themes';
+
+const getTheme = (themeConfig: ThemeConfig) => {
+  const { themeIndex, colorMode } = themeConfig;
+
+  if (themeIndex === 'material') {
+    return colorMode === 'light' ? themeMaterialLight : themeMaterialDark;
+  }
+
+  return colorMode === 'light' ? themeMoniteLight : themeMoniteDark;
+};
+
+export const useTheme = () => {
+  const defaultThemeConfig: ThemeConfig = {
+    themeIndex: 'material',
+    colorMode: 'light',
+  };
+  const [themeConfig, setThemeConfig] = useLocalStorage<ThemeConfig>(
+    'themeConfig',
+    defaultThemeConfig
+  );
+
+  return {
+    theme: getTheme(themeConfig || defaultThemeConfig),
+    themeConfig: themeConfig || defaultThemeConfig,
+    setThemeConfig,
```

Comment:
What if we simplified the returned data:
```
return {
     theme: themeConfig || defaultThemeConfig,
     setTheme: setThemeConfig,
}
```

Then, `getTheme(...)` could be used both in Next.js and in the SDK Demo where it is needed:
```tsx
<AppMoniteProvider theme={getTheme(theme)} />
```

Reply:
fixed